File: app/src/main/java/com/example/annada/eventfinder/model/HereNow.java
package com.example.annada.eventfinder.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by : annada
 * Date : 21/10/2017.
 */

public class HereNow {
    @SerializedName("count")
    @Expose
    private Integer count;
    @SerializedName("summary")
    @Expose
    private String summary;
    @SerializedName("groups")
    @Expose
    private List<Object> groups = null;

    public Integer getCount() {
        return count;
    }

    public void setCount(Integer count) {
        this.count = count;
    }

    public String getSummary() {
        return summary;
    }

    public void setSummary(String summary) {
        this.summary = summary;
    }

    public List<Object> getGroups() {
        return groups;
    }

    public void setGroups(List<Object> groups) {
        this.groups = groups;
    }
}

File: app/src/main/java/com/example/annada/eventfinder/model/HereNow.kt
package com.example.annada.eventfinder.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by : annada
 * Date : 21/10/2017.
 */

class HereNow {
    @SerializedName("count")
    @Expose
    var count: Int? = null
    @SerializedName("summary")
    @Expose
    var summary: String? = null
    @SerializedName("groups")
    @Expose
    var groups: List<Any>? = null
}
